### ISSUE TITLE
error on shoot with crude longbow

### ISSUE THREAD

**Issue by dann1kid** [Zarkonnen/VSRangedWeapons#8]
System.InvalidCastException: Не удалось привести тип объекта "RangedWeapons.EntityPoisonProjectile" к типу "Vintagestory.GameContent.EntityProjectile".
   в Vintagestory.GameContent.ItemBow.OnHeldInteractStop(Single secondsUsed, ItemSlot slot, EntityAgent byEntity, BlockSelection blockSel, EntitySelection entitySel)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.HandleHandInteraction(Single dt)
   в Vintagestory.Client.NoObf.SystemMouseInWorldInteractions.OnGameTick(Single dt)
   в Vintagestory.Common.EventManager.TriggerGameTick(Int64 ellapsedMilliseconds, IWorldAccessor world)
   в Vintagestory.Client.NoObf.ClientMain.MainRenderLoop(Single dt)
   в Vintagestory.Client.NoObf.ClientMain.MainGameLoop(Single deltaTime)
   в _esGsESICkihAc8ShXuIoQwZFwP4._lPY9uZeagiMGC6tvSE11dF7naaz(Single )
   в _8UC8FAMhwKY89IbtSRDjsNSApPJ._KbwUAfxjCHyIG0aK445AAA9FHl5(Single )
   в _8UC8FAMhwKY89IbtSRDjsNSApPJ._R0wAt2N0TpONMxGD0HJG3bDrRcI(Single )
   в Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   в System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   в OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 479
   в OpenTK.GameWindow.DispatchRenderFrame() в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 454
   в OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second) в C:\Users\Nexrem\Desktop\transfer\opentk\src\OpenTK\GameWindow.cs:строка 375
   в _7zNxEbLnuy6Bq08TdMSIu9ZbrKf._vvUTh6X5xLRNnjtX2xSE7z8yMeh(_SUsdg1ytavxaRcNNkhfDrODGhGP , String[] )
   в _yA4PJazdKq1OMm8H6js6v5AhQB._vvUTh6X5xLRNnjtX2xSE7z8yMeh(ThreadStart )

**Comment by Zarkonnen** [comment]
`Could not cast object type 'RangedWeapons.EntityPoisonProjectile' to type 'Vintagestory.GameContent.EntityProjectile'.`

Noted. Probably caused by a game update.

**Comment by Zarkonnen** [comment]
Crude longbows don't seem to be in the game, is it possible they are added by a mod?

**Comment by dann1kid** [comment]
Yes, modded server. If it not yours i dont know what is the trouble

**Comment by Zarkonnen** [comment]
Ranged Weapons is currently incompatible with whatever mod adds the Crude Longbow, basically.
